tests/e2e/iks-cluster

```diff
@@ -1014,6 +1042,17 @@ else
 fi
 
 exportKubeConfig $cluster_name; rc=$?
+
+
```

Comment:
Could you please add a comment here, that you are trying to revert to original addon here, if I am understanding correctly